package.json

```diff
@@ -20,18 +20,16 @@
     "node": ">=6.0.0"
   },
   "scripts": {
-    "clean": "rimraf dist && rimraf dist-esm && rimraf dist-test",
+    "clean": "rimraf dist && rimraf dist-esm",
     "lint": "tslint --project tsconfig.json",
     "format": "prettier --write --config .prettierrc.json \"src/**/*.ts\"",
     "check-format": "prettier --list-different --config .prettierrc.json \"src/**/*.ts\"",
     "compile": "echo Using TypeScript && tsc --version && tsc --pretty",
     "docs": "typedoc --excludePrivate --exclude \"**/test/**\" --mode file --out ./dist/docs ./src",
     "pack": "rollup -c",
     "build": "npm run clean && npm run check-format && npm run lint && npm run compile && npm run docs && npm run pack",
-    "test": "mocha -r ./src/test/common/setup.ts ./dist-test/index.js --timeout 100000 -i",
-    "test-ts": "set TS_NODE_COMPILER_OPTIONS={\"module\":\"commonjs\"} && mocha -r ts-node/register -r ./src/test/common/setup.ts ./src/test/**/*.spec.ts --recursive --timeout 100000 -i -g .*ignore.js",
-    "test-browser": "karma start ./karma.config.js --single-run",
-    "tslint-check": "tslint-config-prettier-check ./tslint.json"
+    "test": "cross-env TS_NODE_COMPILER_OPTIONS='{\"module\":\"commonjs\"}' mocha -r ts-node/register -r ./src/test/common/setup.ts ./src/test/**/*.spec.ts --recursive --timeout 100000 -i -g .*ignore.js",
```

Comment:
I made this cross platform happy with `cross-env`and removed any separate build targets for test. Only one test command now.